Documentation/installation/kind.rst

```diff
@@ -48,36 +48,30 @@ Then, install Cilium release via Helm:
 
 .. note::
 
-   To enable Cilium's kube-proxy replacement (:ref:`kubeproxy-free`), kind nodes
-   need to have their own cgroup namespaces, and are different from the cgroup namespace
+   To enable Cilium's Socket LB (:ref:`kubeproxy-free`), cgroup v2 needs to be
+   enabled, and Kind nodes need to run in separate `cgroup namespaces
+   <https://man7.org/linux/man-pages/man7/cgroup_namespaces.7.html>`__,
+   and these namespaces need to be different from the cgroup namespace
    of the underlying host so that Cilium can attach BPF programs at the right
-   cgroup hierarchy. To verify this, run the following commands inside
-   respective kind nodes, and ensure that the cgroup values are different.
+   cgroup hierarchy. To verify this, run the following commands, and ensure
+   that the cgroup values are different:
 
    .. code-block:: shell-session
 
-      $ docker exec -it kind-control-plane bash
-      $ ls -al /proc/self/ns/cgroup
+      $ docker exec kind-control-plane ls -al /proc/self/ns/cgroup
       lrwxrwxrwx 1 root root 0 Jul 20 19:20 /proc/self/ns/cgroup -> 'cgroup:[4026532461]'
 
-      $ docker exec -it kind-worker bash
-      $ ls -al /proc/self/ns/cgroup
+      $ docker exec kind-worker ls -al /proc/self/ns/cgroup
       lrwxrwxrwx 1 root root 0 Jul 20 19:20 /proc/self/ns/cgroup -> 'cgroup:[4026532543]'
 
-   If you've deployed a kind cluster on an environment where kind nodes don't
-   have their own cgroup namespaces, then cgroup v2 needs to be enabled by
-   setting the kernel ``systemd.unified_cgroup_hierarchy=1`` parameter.
```

Comment:
Revert the accidental deletion of the instructions to enable cgroups v2. As you mentioned, they are systemd specific so we can call that out.